National Library of Norway.js

```diff
@@ -178,27 +297,99 @@ var testCases = [
 				"ISBN": "9788291375052",
 				"libraryCatalog": "National Library of Norway",
 				"numPages": "176",
-				"place": "Lesja",
 				"publisher": "Bondekvinnelaget",
 				"shortTitle": "Mat frå gard og grend",
-				"url": "http://urn.nb.no/URN:NBN:no-nb_digibok_2008030304011",
+				"url": "https://urn.nb.no/URN:NBN:no-nb_digibok_2008030304011",
 				"attachments": [],
 				"tags": [
 					"Foods",
 					"Gudbrandsdalen",
 					"History",
-					"Kokebøker",
 					"Lesja",
 					"Mattradisjoner",
 					"Merkedager",
 					"Preservation",
-					"Religiøse fester"
+					"Religiøse fester",
```

Comment:
these are a lot of tags... are they actually useful?

Reply:
Not sure anymore. The list used to be quite short (probably controlled indexing), but now it seems to be more automatic extraction. I’m removing it.